content/docs/codebase-overview.md

```diff
@@ -149,78 +149,80 @@ var ReactHostComponent = {
 
-### Fiber Reconciler {#fiber-reconciler}
+### Reconciliador Fiber {#fiber-reconciler}
 
-The "fiber" reconciler is a new effort aiming to resolve the problems inherent in the stack reconciler and fix a few long-standing issues. It has been the default reconciler since React 16.
+El reconciliador "fiber" es un nuevo esfuerzo dedicado a resolver los problemas inherentes al reconciliador de pila y arreglar algunos issues. Ha sido el reconciliador por defecto desde React 16.
 
-Its main goals are:
+Sus objetivos principales son:
 
-* Ability to split interruptible work in chunks.
-* Ability to prioritize, rebase and reuse work in progress.
-* Ability to yield back and forth between parents and children to support layout in React.
-* Ability to return multiple elements from `render()`.
-* Better support for error boundaries.
+// TODO: revizar con atención estos puntos
+* Habilidad de dividir trabajo interrumplible en partes.
+* Habilidad de priorizar, y reusar trabajo en progreso.
+* Habilidad para moverse entre padres e hijos para soportar maquetación en React.
+* Habilidad para retornar múltiples elementos desde el método `render()`.
+* Mejor soporte a error boundaries.
 
-You can read more about React Fiber Architecture [here](https://github.com/acdlite/react-fiber-architecture) and [here](https://medium.com/react-in-depth/inside-fiber-in-depth-overview-of-the-new-reconciliation-algorithm-in-react-e1c04700ef6e). While it has shipped with React 16, the async features are not enabled by default yet.
+Puedes leer más acerca de la Arquitectura de React Fiber [aquí](https://github.com/acdlite/react-fiber-architecture) y [aquí](https://medium.com/react-in-depth/inside-fiber-in-depth-overview-of-the-new-reconciliation-algorithm-in-react-e1c04700ef6e). Como el soporte comenzó desde React 16, las funcionalidades asíncronas no se han habilitado aún.
 
-Its source code is located in [`packages/react-reconciler`](https://github.com/facebook/react/tree/master/packages/react-reconciler).
+Su código fuente esta ubicado en [`packages/react-reconciler`](https://github.com/facebook/react/tree/master/packages/react-reconciler).
 
-### Event System {#event-system}
+### Sistema de Eventos {#event-system}
 
-React implements a synthetic event system which is agnostic of the renderers and works both with React DOM and React Native. Its source code is located in [`packages/events`](https://github.com/facebook/react/tree/master/packages/events).
+React implementa un sistema de eventos sintético que es agnostico de los renderizadores y funciona con React DOM y React Native. Su código fuente esta localizado en [`packages/events`](https://github.com/facebook/react/tree/master/packages/events).
 
-There is a [video with a deep code dive into it](https://www.youtube.com/watch?v=dRo_egw7tBc) (66 mins).
+Hay un [video con una muestra a profundidad del código](https://www.youtube.com/watch?v=dRo_egw7tBc) (66 mins).
 
-### What Next? {#what-next}
+### Que sigue? {#what-next}
```

Comment:
```suggestion
### Qué sigue? {#what-next}
```

Reply:
This change was also suggested by @icarlossz, so it is already done.